entry/streamer-entry

```diff
@@ -0,0 +1,211 @@
+  local ice_port_begin=${ICE_PORT_BEGIN:-"50000"}
+  local ice_port_end=${ICE_PORT_END:-"50999"}
+
+  local ipc files
+  if [[ "$k8s" == "true" ]]; then
+    ipc="/conn"
+    wdir="/conn"
+    nid=0
+    ips=${K8S_ENV_OWT_SERVER_IP:-owt}
+    port=${K8S_ENV_OWT_SERVER_PORT:-8095}
+    portOpt=" -p $port"
+    k8sOpt="--k8s 1"
+    local GPU_NODES=${GPU_NODES:-$(find /dev/dri -iname "renderD*" | wc | awk '{print $1}')}
+    local INS_ON_NODE="$(expr $id % $GPU_NODES + 128)"
+    dev="/dev/dri/renderD$INS_ON_NODE"
+    if [[ "$k8s_stateless" == "true" ]]; then
+        wait_file /conn/stateless/id
+        dev="/dev/dri/renderD128"
+        IFS="," read -ra arr <<< "$(cat /conn/stateless/id)"
+        serverid=${arr[0]}
+        clientid=${arr[1]}
+        echo wait for icr container start up...
+        sleep 10
+    fi
+  else
+    ipc="/opt/workdir/ipc"
+    nid=$id
+  fi
+  if [[ $VIRTUAL_INPUT_NUM -gt 0 ]]; then
+    for ((i=0;i<$VIRTUAL_INPUT_NUM; i++)); do
+      if [[ "$k8s" == "true" ]]; then
+        files="input-pipe-$i"
+      elif [[ $user_count -gt 1 ]]; then
+        files="input-pipe${id}-$uid-$i"
+      else
+        files="input-pipe${id}-$i"
+      fi
+    done
+  fi
+  for f in $files; do
+    wait_file "${ipc}/${f}"
+  done
+
+  local opts="-s $ips  $portOpt $k8sOpt -d $dev --width $w --height $h -n $nid --server-peer-id $serverid --client-peer-id $clientid --workdir $wdir"
+
+  if [[ $user_count -gt 1 ]]; then
+      opts="$opts --enable-multi-user 1 --user $uid"
+  fi
+
+  if [ "$cqp" == "true" ]; then
+      opts="$opts --video-rc cqp"
+  fi
+
+  if [ "$y" == "y" ]; then
+      if [[ $user_count -gt 1 ]]; then
+          iIP=`cat "/opt/workdir/icr/icr"$id-$uid`
+      else
+          iIP=`cat "/opt/workdir/icr/icr"$id`
+      fi
+      opts="$opts --icr-ip $iIP"
+  fi
+
+  if [[ "$k8s" == "true" ]]; then
+      opts="$opts --icr-ip 127.0.0.1"
+  fi
+
+  if [ "$measure_latency" == "true" ]; then
+      opts="$opts --measure-latency 1"
+  fi
+
+  if [[ "$hevc_enable" == "true" ]]; then
+      opts="$opts --codec h265"
+  fi
+
+  if [[ "$icr_start_immediately" == "true" ]]; then
+      opts="$opts --icr-start-immediately 1"
+  fi
+
+  if [ -n "$coturn_ip" ]; then
+    opts="$opts --coturn-ip $coturn_ip"
+  fi
+
+  if [ $ice_port_begin -ge 1 ]; then
+    opts="$opts --ice-port-min $ice_port_begin"
+  fi
+
+  if [ $ice_port_end -ge 1 ]; then
+    opts="$opts --ice-port-max $ice_port_end"
+  fi
+
+  dbg "opts:$opts"
+  if [[ "$privileged_enable" == "true" ]]; then
+      dbg "mount /sys/kernel/debug"
+      mount -t debugfs none /sys/kernel/debug
+      chmod 0755 /sys/kernel/debug
+      chmod 0222 /sys/kernel/debug/tracing/trace_marker
+      chmod 0755 /sys/kernel/debug/tracing
+  fi
+
+  while true; do
+     DISABLE_DYNAMIC_RESOLUTION=1 /opt/bin/ga-server-periodic $opts /opt/etc/ga/server.desktop.webrtc.conf
+     echo "ga server exit, may due to AIC is not ready, try it 2s later"
+     sleep 2
+  done
```

Comment:
Because this mentions cloud and k8s, I assume it's for Kubernetes.

However, that's not how you're supposed to do dependencies.  See e.g: https://stackoverflow.com/questions/56935239/how-to-configure-pod-initialization-in-a-specific-order-in-kubernetes